apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -454,7 +454,7 @@ export const IndictmentCount: React.FC<React.PropsWithChildren<Props>> = (
       </Box>
       <Box marginBottom={2}>
         <InputMask
-          mask={'**-*99'}
+          mask={[/[A-Z]/i, /[A-Z]/i, /[A-Z]|[0-9]/i, /[0-9]/, /[0-9]/]}
```

Comment:
Is it certain that only the middle character can be both a letter and a digit?

Reply:
Yes, I believe so.. 

The vehicle registration number is either LETTER LETTER DIGIT DIGIT DIGIT or LETTER LETTER LETTER DIGIT DIGIT.